react/features/recording/actions.any.ts

```diff
@@ -367,3 +379,62 @@ export function stopLocalVideoRecording() {
         type: STOP_LOCAL_RECORDING
     };
 }
+
+/**
+ * Displays the notification suggesting to start the recording.
+ *
+ * @returns {void}
+ */
+export function showStartRecordingNotification() {
+    return (dispatch: IStore['dispatch'], getState: IStore['getState']) => {
+        const state = getState();
+        const { recordings } = state['features/base/config'];
+        const { suggestRecording } = recordings || {};
+        const recordButtonProps = getRecordButtonProps(state);
+        const isAlreadyRecording = isRecordingRunning(state);
+
+        if (!suggestRecording
+            || isAlreadyRecording
+            || !recordButtonProps.visible
+            || recordButtonProps.disabled) {
+            return;
+        }
+
+        dispatch(showNotification({
+            titleKey: 'dialog.startRecording',
+            uid: START_RECORDING_NOTIFICATION_ID,
+            customActionNameKey: [ 'dialog.startRecording' ],
+            customActionHandler: [ () => {
+                const isModerator = isLocalParticipantModerator(state);
+                const { recordingService } = state['features/base/config'];
+
+                const canBypassDialog = isModerator
+                    && recordingService?.enabled
+                    && isJwtFeatureEnabled(state, 'recording', true);
+
+                if (canBypassDialog) {
+                    const options = {
+                        'file_recording_metadata': {
+                            share: true
+                        }
+                    };
+
+                    const { conference } = state['features/base/conference'];
+                    const { autoTranscribeOnRecord } = state['features/base/config'].transcription || {};
```

Comment:
Shoudln't the default for this be true?

Reply:
you mean the default on the fallback object right?

Reply:
Yep. Here and anywhere else we rely on this thing. Maybe a small selector could be helpful?